sgp4/exporter.py

```diff
@@ -62,18 +62,26 @@ def export_tle(satrec):
         append(str(satrec.satnum).zfill(5) + " ")
 
     # Add the inclination (deg)
+    if not 0 <= satrec.inclo <= pi:
+        raise ValueError("Inclination must be between 0 and pi, got %f", satrec.inclo)
```

Comment:
Let’s try `%r` instead of `%f` for these value checks, in case anyone is ever using the old Python satrec and inserts a value that’s not a float.

Reply:
Done!